### PR TITLE
Mm 6.0 selinux

### PR DESCRIPTION
**Pull request by alviteri**

### CODE REVIEW

**Comment by humberos** [comment]
@alviteri just remove timekeep_app denial since it should be removed.
timekeep_app should be system_app for property_socket permission.
-> https://github.com/sonyxperiadev/device-sony-sepolicy/pull/8

**Comment by alviteri** [comment]
OK , thanks @humberos will do tomorrow

File: netmgrd.te [hunk 1]
```diff
@@ -37,6 +37,8 @@ allow netmgrd system_file:file x_file_perms;
 
 allow netmgrd self:socket create_socket_perms;
 
+allow netmgrd sysfs_pronto:file { read open };
+
```
Comment (humberos) [inline netmgrd.te]: Use r_file_perms instead of { read open }

File: rild.te [hunk 2]
```diff
@@ -11,5 +11,9 @@ netmgr_socket(rild);
 # Rule for RILD to talk to peripheral manager
 use_per_mgr(rild);
 
+allow rild sysfs_pronto:file { read open };
```
Comment (humberos) [inline rild.te]: same here

File: system_server.te [hunk 3]
```diff
@@ -15,6 +15,8 @@ use_per_mgr(system_server);
 
 allow system_server system_app_data_file:dir r_dir_perms;
 
+allow system_server sysfs_pronto:file { read open };
```
Comment (humberos) [inline system_server.te]: and here

**Comment by alviteri** [comment]
fixed

**Comment by humberos** [comment]
great!
thank you!

File: surfaceflinger.te [hunk 4]
```diff
@@ -1 +1,3 @@
 rw_dir_file(surfaceflinger, sysfs_graphics)
+
+allow surfaceflinger sysfs:file w_file_perms;
```
Comment (AdFad666) [inline surfaceflinger.te]: We should determine which sysfs and create a label for it, this allows to write to **all** sysfs

**Comment by alviteri** [comment]
ok ... updated ... surfaceflinger can be fixed later

**Pull request merged**